components/num-util/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "icu-num-util"
+description = "Low-level tools for dealing with numbers"
+version = "0.0.1"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "../../LICENSE"
+categories = ["internationalization"]
+include = [
+    "src/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+[dependencies]
+smallvec = "1.4.0"
+num-traits = "0.2.12"
+num-integer = "0.1.43"
+static_assertions = "1.1.0"
+more-asserts = "0.2.1"
```

Comment:
By "unstable" do you mean those with major version 0?

- `num-traits` and `num-integer`, which are in the meta-crate `num`, are my attempt at generalizing the `from` methods for FixedDecimal.  If we remove them, you will still be able to construct a FixedDecimal from hard-coded types like `isize`, but not from `BigUint` or other types.  My guess is that some of the `num` crate might end up in the stdlib one day, but I do not know their plans.  I have no problem removing these if it's better without them.
- `more-asserts` was so I could get `>` and `<` asserts.  I think removing this one would make code quality go down, and it doesn't affect what we ship since these are only used in debug mode, so I vote to keep it.

Reply:
I see, thank you.

> num-traits and num-integer, which are in the meta-crate num, are my attempt at generalizing the from methods for FixedDecimal. If we remove them, you will still be able to construct a FixedDecimal from hard-coded types like isize, but not from BigUint or other types.

My first feeling is that it's a bit premature to aim for compatibility with a custom type like `biguint`.

I'd like to suggest that we first write this for just primitive types, potentially using the model such as https://github.com/zbraniecki/pluralrules/blob/master/intl_pluralrules/src/operands.rs#L183-L186 and then we evaluate scaling it up to handle non-standard types such as `biguint` or some other version of it if another library gain traction.